packages/astro/src/core/build/plugins/plugin-content.ts

```diff
@@ -0,0 +1,288 @@
+import { normalizePath, type Plugin as VitePlugin } from 'vite';
+import fsMod from 'node:fs';
+import { createHash } from 'node:crypto';
+import { addRollupInput } from '../add-rollup-input.js';
+import { type BuildInternals } from '../internal.js';
+import type { AstroBuildPlugin } from '../plugin.js';
+import type { StaticBuildOptions } from '../types.js';
+import { generateContentEntryFile, generateLookupMap } from '../../../content/vite-plugin-content-virtual-mod.js';
+import { joinPaths, removeFileExtension, removeLeadingForwardSlash } from '../../path.js';
+import { fileURLToPath } from 'node:url';
+import { type ContentLookupMap, hasContentFlag } from '../../../content/utils.js';
+import { CONTENT_RENDER_FLAG, PROPAGATED_ASSET_FLAG } from '../../../content/consts.js';
+import { copyFiles } from '../static-build.js';
+import pLimit from 'p-limit';
+import { extendManualChunks } from './util.js';
+import { isServerLikeOutput } from '../../../prerender/utils.js';
+import { encodeName } from '../util.js';
+
+const CONTENT_CACHE_DIR = './content/';
+const CONTENT_MANIFEST_FILE = './manifest.json';
+// IMPORTANT: Update this version when making significant changes to the manifest format.
+// Only manifests generated with the same version number can be compared.
+const CONTENT_MANIFEST_VERSION = 0;
```

Comment:
smart!

Reply:
Thanks! Seemed wise to have a kill-switch if needed.